tools/tests/com.liferay.ide.server.core.tests/src/com/liferay/ide/server/core/tests/ServerManagerTests.java

```diff
@@ -247,6 +249,8 @@ public void testInstallUpdateUninstallApplication() throws Exception
 
         final NullProgressMonitor npm = new NullProgressMonitor();
 
+        assertEquals( "Expected the remote connection's status should be alive", true, service.isAlive() );
```

Comment:
Move this assertEquals(...) into here because of "service.isAlive()" going to get response from server according to 
server-manager-web-6.2.0.3.war. But this war file could not be deploy in Liferay 7. It cause test fault.